### PR TITLE
Edits to Configuring network subnets

### PR DESCRIPTION
**Pull request by ahardin-rh**
@sjug Thanks for raising this issue. PTAL.

### CODE REVIEW

File: modules/configuring-network-subnets.adoc [hunk 1]
```diff
@@ -33,12 +33,12 @@ $ cd new-install
...
-      clusterNetworks:
+      clusterNetwork:
         - cidr: 10.128.0.0/14
-          hostSubnetLength: 9
+          hostPrefix: 9
       machineCIDR: 10.0.0.0/16
-      serviceCIDR: 172.30.0.0/16
-     type: OpenshiftSDN
+      serviceNetwork: 172.30.0.0/16
+     networkType: OpenshiftSDN
```
Comment (sjug) [inline modules/configuring-network-subnets.adoc]: One more space indent (should be at same depth as `clusterNetwork`, `machineCIDR` and `serviceNetwork`)

Comment (ahardin-rh) [reply modules/configuring-network-subnets.adoc]: @sjug Fixed!

**Comment by sjug** [comment]
I'm just wondering if we should add the `apiVersion: v1beta4` for this snippet as well?

The previous fields are from `v1beta3` and are now deprecated. If we just keep the docs up to date then we assume that it's the latest API version?

**Comment by ahardin-rh** [comment]
@sjug We need to keep the docs up to date. If the API version changes so frequently, then do we want to state it explicitly here?

**Comment by sjug** [comment]
Good point, lets not change/add it. I was just thinking it gives people a reference point of where in the yaml they are, but they can figure it out.

**Comment by jboxman** [comment]
Also, `hostPrefix` should be `23`: https://github.com/openshift/openshift-docs/pull/14224#discussion_r270504697

**Comment by ahardin-rh** [comment]
@jboxman Good catch. Updated! Merging!

**Pull request merged**